components/Paginated/Paginated.module.scss

```
@@ -0,0 +1,96 @@
+@import 'colors','fonts','layout';
+
+.paginated-container {
+  align-items: center;
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  margin: 10px 50px;
+  max-height: 43px;
+  padding: 20px auto;
+  position: fixed;
```

```suggestion
  position: absolute;
  right: 0;
```
but you have also his father element relative. Read this [ref ](https://css-tricks.com/absolute-positioning-inside-relative-positioning/)

---

ok

---

only these?